### PR TITLE
Fix Errcheck issues in server/channels/api4/export.go

### LINKED ISSUE #28385: Fix errcheck issues in server/channels/api4/export.go
At Mattermost, we are committed to maintaining a high standard of code quality, reliability, and security. To further enhance this, we are launching a community campaign to integrate the [`errcheck`](https://github.com/kisielk/errcheck) linter across the Mattermost codebase.

This Help Wanted ticket is about fixing the linter issues reported by `errcheck` in `channels/api4/export.go`.

### What is `errcheck`?
`errcheck` is a tool that automatically identifies places in Go code where errors are returned but not checked. In Go, error handling is a critical part of writing robust applications, and ignoring returned errors can lead to bugs, crashes, or unintended behaviour. By integrating `errcheck` into the codebase, we can ensure that error handling is consistently and properly managed across the project.

### Tasks
- Remove the exception for `channels/api4/export.go` from the exceptions list for `errcheck` in `server/.golangci.yml`.
- Run `make golangci-lint` to get a list of all linter errors.
- Fix the errors by properly handling them.
- Run `make golangci-lint` again to ensure all errors are fixed.

#### Okay, but what does "properly handling" mean?
In general, errors should be returned to the caller. Non critical errors should be loggged, and function execution should continue. There are some exceptions to this rule, such as when the error is expected and should be ignored or when the error is handled in a different way. In such cases, a comment should be added to explain why the error is ignored.

In tests, errors should be checked using `require.NoError(t, err)` for errors of type `error` or `require.Nil(t, err)` for errors of type `**model.AppError`.
If an error is expected, it should be checked using `require.Error(t, err)` for errors of type `error` or `require.NotNil(t, err)` for errors of type `**model.AppError`.

Use `err` as the variable name for errors and `appErr` for `*model.AppError`. Avoid using variable names like `err1`, `err2`, etc. See [the Mattermost style guide](https://developers.mattermost.com/contribute/more-info/server/style-guide/#error-variable-names) for more information.

https://github.com/mattermost/mattermost/pull/28364 and https://github.com/mattermost/mattermost/pull/28376 can be used as examples of how to fix `errcheck` issues in the `api4` package.

-------------

If you're interested, please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on the Mattermost Community server, where you can discuss questions with community members and the Mattermost core team.
For technical advice or questions, please [join our "Developers Server" community channel](https://community.mattermost.com/core/channels/developers-server).
You can also reach out to @hanzei either on this ticket or on [the Mattermost Community server](https://community.mattermost.com).

For new contributors, please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

If this ticket is already taken, please check out our other [Help Wanted tickets](https://github.com/mattermost/mattermost/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+errcheck+label%3A%22Up+For+Grabs%22) of this campaign.

Thank you for your contributions to Mattermost!

----------------

JIRA: MM-60752

### PR DESCRIPTION
**Pull request by diamant3**
#### Ticket Link

Fixes #28385 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: server/channels/api4/export.go [hunk 1]
```diff
@@ -5,12 +5,12 @@ package api4
...
 	"encoding/json"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	"github.com/mattermost/mattermost/server/v8/channels/audit"
 	"net/http"
 	"path/filepath"
 	"time"
-
-	"github.com/mattermost/mattermost/server/public/model"
-	"github.com/mattermost/mattermost/server/v8/channels/audit"
```
Comment (hanzei) [inline server/channels/api4/export.go:13]: Can you please restore the previous order of imports?

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the contribution :+1:

One request in the comments

File: server/channels/api4/export.go (hunk 1 above)
Comment (diamant3) [reply server/channels/api4/export.go:13]: Done! ❤️

File: server/channels/api4/export.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"net/http"
 	"path/filepath"
 	"time"
-
```
Comment (hanzei) [inline server/channels/api4/export.go:11]: Almost. Please add this empty line back.

**Review by hanzei — CHANGES_REQUESTED** [review]

File: server/channels/api4/export.go (hunk 2 above)
Comment (diamant3) [reply server/channels/api4/export.go:11]: Done! ❤️

**Review by hanzei — APPROVED** [review]
Awesome :rocket:

**Review by JulienTant — APPROVED** [review]
Thanks for your contribution @diamant3

File: server/channels/api4/export.go [hunk 3]
```diff
@@ -9,6 +9,7 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
 	"github.com/mattermost/mattermost/server/public/model"
 	"github.com/mattermost/mattermost/server/v8/channels/audit"
```
Comment (hanzei) [inline server/channels/api4/export.go:14]: ordering 
```suggestion
	"github.com/mattermost/mattermost/server/public/model"
	"github.com/mattermost/mattermost/server/public/shared/mlog"
	"github.com/mattermost/mattermost/server/v8/channels/audit"
```

**Comment by hanzei** [comment]
Thanks for the contribution @diamant3 :tada:

**Pull request merged**